atom.sh

```diff
@@ -1,6 +1,6 @@
 # Configure atom sync
 
-apm install sync-settings
+apm install sync-settings api-blueprint-preview change-case css-hyperclick docblockr git-blame git-plus git-time-machine hyperclick js-hyperclick language-api-blueprint language-docker linter linter-eslint linter-ui-default nuclide nuclide-format-js pigments sort-lines var-that-color
```

Comment:
the point here is that you install sync settings and then sync settings installs everything else